### PR TITLE
feat: page profile

### PR DESCRIPTION
**Pull request by DenisMedeirosSDK**
#10 
Create struct basic for profile page, awaiting new components create.

BUGS: Necessary change component HEADER, input not working with CSS : "box-sizing: border-box;"

image

### CODE REVIEW

File: src/config/routes/index.js [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
-export const routes = [
-  {
-    path: '/',
-    component: HomePage,
-  },
+import { HomePage, Profile } from '../../pages'
+
+export const routes = [
+  {
+    path: '/',
```
Comment (pablobrodt) [inline src/config/routes/index.js:5]: Could create a constant to store all application routes.

Comment (pablobrodt) [reply src/config/routes/index.js:5]: could be in config/routes

File: src/components/StreamerProfile/index.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import picProfile from "../../assets/jeffprofile.png";
+
+import styles from "./styles.module.scss";
+
+export function StreamerProfile({
```
Comment (pablobrodt) [inline src/components/StreamerProfile/index.js]: Why streamer profile? isnt going to be the profile for anyone in the future?

File: src/config/routes/index.js (hunk 1 above)
Comment (DenisMedeirosSDK) [reply src/config/routes/index.js:5]: I followed the pattern already created, same as the home page

File: src/components/StreamerProfile/index.js (hunk 2 above)
Comment (DenisMedeirosSDK) [reply src/components/StreamerProfile/index.js]: Jeff said it's only for the streamer community, and any twitch account is a streamer account

**Comment by DenisMedeirosSDK** [comment]
Update page profile 

#11 

image

image

**Pull request closed without merging**